cf_units/meta.yaml

```diff
@@ -11,7 +11,7 @@ build:
 
 requirements:
     build:
-        - python
+        - python <3
```

Comment:
Yes. And any future work to allow py3 support would need to be a new build number.